src/Arcus.Scripting.ApiManagement/Arcus.Scripting.ApiManagement.psm1

```diff
@@ -173,3 +173,37 @@ function Import-AzApiManagementOperationPolicy {
 
     . $PSScriptRoot\Scripts\Import-AzApiManagementOperationPolicy.ps1 -ResourceGroup $ResourceGroup -ServiceName $ServiceName -ApiId $ApiId -OperationId $OperationId -PolicyFilePath $PolicyFilePath
 }
+
+<#
+ .Synopsis
+  Sets the authentication keys in Azure API Management.
+
+ .Description
+  Sets the authentication header/query parameter on an API in Azure API Management.
+
+ .Parameter ResourceGroup
+  The resource group containing the Azure API Management instance.
+
+ .Parameter ServiceName
+  The name of the Azure API Management instance located in Azure.
+  
+ .Parameter ApiId
+  The ID to identify the API running in Azure API Management.
+
+ .Parameter ApiKeyHeaderName
+  The name of the header where the subscription key should be set.
+
+ .Parameter ApiQueryParamName
```

Comment:
What if we remove the Api prefix? Thoughts on this?
```suggestion
 .Parameter QueryParamName
```

Reply:
Renamed, resolving.